src/kirby/components/card/card.component.tns.html

```diff
@@ -1,4 +1,4 @@
-<FlexboxLayout (loaded)="onViewLoaded($event)" class="background" [ngClass]="cardSizeClass">
+<FlexboxLayout class="background" [ngClass]="cardSizeClass" [kirbyElevation]="getElevation()">
```

Comment:
Be carefull with method calls vs properties as change detection might have performance impact:
```suggestion
<FlexboxLayout class="background" [ngClass]="cardSizeClass" [kirbyElevation]="elevation">
```